manifests/remote.pp

```diff
@@ -21,4 +23,13 @@
     target  => $config_file,
     content => template('unbound/remote.erb'),
   }
+
+  exec { 'unbound-control-setup':
+    command => "unbound-control-setup -d ${confdir} && \
+		chgrp ${group} ${server_key_file} \
```

Comment:
I'd prefer that these `chgrp` be broken out into file resources for readability if thats not too much trouble.